crates/extricrate/src/lib.rs

```diff
@@ -171,6 +195,21 @@ pub mod dependencies {
         None
     }
 
+    fn mod_to_path(crate_root: &Path, ancestors: &[String], ident: &Ident) -> Option<PathBuf> {
+        let ident = ident.to_string();
+        let mut root_path = crate_root.join("src");
+        root_path.extend(ancestors);
+
+        let file_module = root_path.join(format!("{}.rs", ident));
+        let folder_module = root_path.join(&ident).join("mod.rs");
+        if crate_root.join(&file_module).exists() {
+            return Some(file_module);
+        } else if crate_root.join(&folder_module).exists() {
+            return Some(folder_module);
+        }
+        None
```

Comment:
I would probably make it a `Result` and return an error along the lines of "Could not find source file for module \`bla\`"